change-address/LETTERSPECS/BANNO.CHANGE.ADDR.V1.CFG

```diff
@@ -85,6 +85,17 @@ PO Box List 03: PO BOX
 PO Box List 04: PO. BOX
 PO Box List 05: POST OFFICE BOX
 *
+*
+*! IF ACTIVATING NAME MATCHING AFTER PREVIOUSLY USING THE ADDRESS CHANGE      !
+*! POWERON, WE RECOMMEND REMOVING THE POWERON 'BANNO.DATABASE.CHECK.V1.POW'   !
```

Comment:
You may want to include "uninstalling it for demand use" before deleting it - otherwise, couldn't it possibly cause a fatal during system PowerOn reinstalls?

Reply:
Done